### ISSUE TITLE
Running into timeout issues when downloading Serverless cli

### ISSUE THREAD

**Issue by ogii** [CircleCI-Public/serverless-framework-orb#15]
**Orb Version**
1.0.1

**Describe the bug**
The following curl command to install the Serverless cli in the `setup` command occasionally results in a timeout:

`curl -o- -L https://slss.io/install | bash`

![serverless](https://user-images.githubusercontent.com/640433/132839591-3becce83-76a7-4a7a-bb76-e5489e095304.png)

**To Reproduce**
Use the `circleci/serverless-framework` orb in a build, and run the `setup` command.

**Expected behavior**
The Serverless cli should be installed after running the `setup` command.

**Additional context**
This is an intermittent issue, but I noticed it happened around 10%~15% of the time. 
Adding a retry to the curl command on timeout might address the issue.

**Comment by nshgraph** [comment]
For me this timeout is 80%+ of the time and means th9s orb doesn't work.

The issue seems to be that CircleCI is appearing to be in china to the serverless install script. The timed out log entry is:
`Connection reset by peer in connection to sls-standalone-1300963013.cos.ap-shanghai.myqcloud.com`

From the orb source:
`curl -o- -L https://slss.io/install | bash`

This bash script seems to use myqcloud.com only if it is detected that the caller is in China. And that URL is definitely trying to download from China.

Because the only part of the script that I need is the actual serverless install I have resolved this issue by adding the following to replace `-serverless/setup`

```
curl -o- -L https://slss.io/install | SLS_GEO_LOCATION=us bash 
echo "export PATH=$HOME/.serverless/bin:$PATH" >> "$BASH_ENV"
source "$BASH_ENV"
```

This forces it to use the US source and voila... no timeout.

If CircleCI only builds in the US, then adding this to the orb would probably fix the issue.

**Comment by jp172** [comment]
was happy to have found this workaround a few days ago, but it seems to be timing out for `SLS_GEO_LOCATION=us` as well

<img width="950" alt="slss-timeout" src="https://user-images.githubusercontent.com/7261000/135282779-20bef709-ebf7-4d36-8dff-493871f38602.png">

**Comment by nshgraph** [comment]
Sorry I should have updated. It turned out to be two things: the first is that IFF it gets the script, the script was downloading from China. But also the script itself is being hosted in china (wtf serverless?)

`curl -I -L  https://slss.io/install
HTTP/1.1 301 Moved Permanently
Server: nginx
Date: Wed, 29 Sep 2021 19:47:23 GMT
Content-Type: text/html
Content-Length: 178
Connection: keep-alive
Location: https://tinycc.com/tiny/custom_domain_redirect/slss.io/install

HTTP/1.1 303 See Other
Server: nginx
Date: Wed, 29 Sep 2021 20:21:17 GMT
Content-Type: text/html; charset=utf-8
Connection: keep-alive
X-Powered-By: PHP/7.0.33
X-Frame-Options: sameorigin
X-XSS-Protection: 1; mode=block
X-Robots-Tag: nofollow, noindex
Location: https://sls-standalone-1300963013.cos.ap-shanghai.myqcloud.com/install.sh
Strict-Transport-Security: max-age=31536000;
Content-Security-Policy-Report-Only: default-src https:; script-src https: 'unsafe-eval' 'unsafe-inline'; style-src https: 'unsafe-inline'; img-src https: data:; font-src https: data:; report-uri /csp-report

HTTP/1.1 200 OK
Content-Type: application/x-sh
Content-Length: 3897
Connection: keep-alive
Date: Wed, 29 Sep 2021 20:20:45 GMT
ETag: "d97c0ea9754ead9a0e98f3066793f590"
Last-Modified: Mon, 08 Feb 2021 10:39:46 GMT
Server: tencent-cos
x-cos-hash-crc64ecma: 6200904514863612494
x-cos-request-id: NjE1NGNhOWRfZDU0MTIyMDlfNWMyNl83OTQ5MTA=`

So we can see it is being pulled from  https://sls-standalone-1300963013.cos.ap-shanghai.myqcloud.com/install.sh which can time out because it is in China.

Serverless mirrors its files in the US too so replacing the URL with https://sls-standalone-sv-1300963013.cos.na-siliconvalley.myqcloud.com/install.sh means china doesn't get included.

**Comment by KyleTryon** [comment]
@nshgraph thank you for providing this info. It seems like maybe something we might be looking to serverless to correct but I wanted to know if you ended up finding a proper solution that worked for you long time. I am in a maintenance sprint for this orb, but this is very difficult for me to reproduce. I will add `SLS_GEO_LOCATION=us` as mentioned above but as you also pointed out, this may not be enough.

We can pull from that second URL but I have concerns this will be difficult to keep up to date and based on the age of this issue, likely already is.

**Comment by KyleTryon** [comment]
Hello,
Thank you all for reporting this issue. We have just released version 2.0.0 which should resolve this problem. Please try the latest version of the orb and let us know if you experience any issues.

https://github.com/CircleCI-Public/serverless-framework-orb/releases/tag/v2.0.0
https://circleci.com/developer/orbs/orb/circleci/serverless-framework
